modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectStateFlowLocalServiceImpl.java

```diff
@@ -183,6 +183,48 @@ public ObjectStateFlow updateDefaultObjectStateFlow(
 		return null;
 	}
 
+	public void updateObjectStateFlowTransitions(
```

Comment:
@G-Sa rename it to `updateObjectStateFlowObjectstateTransitions` to follow the same pattern from `deleteObjectStateFlowObjectStateTransitions`